dashboard/test/ui/features/star_labs/signin_callout2.feature

```diff
@@ -2,11 +2,11 @@ Feature: Viewing and dismissing the login callout
 
   @no_mobile
   Scenario: Clicking anywhere should dismiss the login reminder
-    Given I am on "http://studio.code.org/courses/20-hour/units/1/lessons/9/levels/1?noautoplay=true"
+    Given I am on "http://studio.code.org/courses/coursea-2020/units/1/lessons/5/levels/3?noautoplay=true"
```

Comment:
For my understanding, why couldn't we use allthethings for these?

Reply:
The callout is only shown for CSF or CSC curriculum ([code](https://github.com/code-dot-org/code-dot-org/blob/047507997727679c80144c33380ee6c89a403008/dashboard/app/models/unit.rb#L1467). All the Things is neither of those, so this test won't work in that curriculum. The best option would be to add a UI test specific curriculum, such as the one we have for [CSA](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/config/courses/ui-test-csa-family-script.course), but I didn't really have the cycles to create one. :/ I briefly tried, hit an issue, then put it down and went with what we've already got in this test.

Reply:
haha yeah that sounds like a pain. Thanks for the explanation!

Reply:
thanks for sharing the details. I may be trying to get all our UI tests onto ui-test-only units soon so this is helpful!